src/linear_algebra/affine_space/combination.lean

```diff
@@ -229,6 +229,30 @@ lemma affine_combination_vsub (w₁ w₂ : ι → k) (p : ι → P) :
   s.affine_combination p w₁ -ᵥ s.affine_combination p w₂ = s.weighted_vsub p (w₁ - w₂) :=
 by rw [←affine_map.linear_map_vsub, affine_combination_linear, vsub_eq_sub]
 
+lemma affine_combination_attach (s : finset P) (w : P → k) :
```

Comment:
Can't this be generalised to any `f : s → P`? The name suggests this at least.

Reply:
Hmm, I think it can indeed be generalised as long as we assume `f` is injective.

So I guess the statement would become something like:
```lean
lemma affine_combination_attach (s : finset P) (w : P → k) (f : s → P) (hf : function.injective f) :
  s.attach.affine_combination f (w ∘ f) = (image f univ).affine_combination id w :=
```

I'll see about adding this later this afternoon.

Reply:
Now done.